src/main/java/org/checkstyle/autofix/recipe/RedundantImport.java

```diff
@@ -60,7 +60,8 @@ private final class RemoveRedundantImportsVisitor extends JavaIsoVisitor<Executi
         private Path sourcePath;
 
         @Override
-        public J.CompilationUnit visitCompilationUnit(J.CompilationUnit cu, ExecutionContext ctx) {
+        public J.CompilationUnit visitCompilationUnit(J.CompilationUnit cu,
+                                                      ExecutionContext executionContext) {
```

Comment:
Great to see the addition of `parameterName` to `ExecutionContextParameterName` has helped consistency here. 🚀